packages/lambda-xyz/src/__test__/wmts.capability.test.ts

```diff
@@ -1,142 +1,120 @@
+o.spec('WmtsCapabilities', () => {
+    const apiKey = 'secret1234';
+    const tileSet = new FakeTileSet('aerial', Epsg.Google);
+    const tileSetImagery = new FakeTileSet('01E7PJFR9AMQFJ05X9G7FQ3XMW', Epsg.Google);
 
-o.spec('wmts', () => {
-    o.beforeEach(() => {
-        for (const name of TileSetNames) {
-            const tileSet = new TileSet(name, Epsg.Google);
-            addTitleAndDesc(tileSet);
-            TileSets.set(tileSet.id, tileSet);
-        }
-    });
+    o('should build capability xml for tileset and projection', () => {
+        const wmts = new WmtsCapabilities('https://basemaps.test', Provider, [tileSet], apiKey);
 
-    o.afterEach(() => {
-        TileSets.clear();
-    });
+        const raw = wmts.toVNode();
+        const serviceId = raw.find('ows:ServiceIdentification');
 
-    o('should build capabiltiy xml for tileset and projection', () => {
-        const req = mockRequest('/v1/tiles/aerial@beta/3857/WMTSCapabilities.xml', 'get', {
-            [HttpHeader.ApiKey]: 'secret1234',
-        });
-        const raw = buildWmtsCapabilityToVNode(
-            'https://basemaps.test',
-            req,
-            Provider,
-            TileSets.get('aerial@beta_3857')!,
-        )!;
-
-        const serviceId = raw.find('ows:ServiceIdentification')!;
-
-        o(serviceId.find('ows:Abstract')!.textContent).equals('the description');
-        o(serviceId.find('ows:Title')!.textContent).equals('the title');
-
-        o(listTag(raw, 'TileMatrixSetLink')).deepEquals([
-            '<TileMatrixSetLink>\n' +
-                '  <TileMatrixSet>GoogleMapsCompatible</TileMatrixSet>\n' +
-                '</TileMatrixSetLink>',
-        ]);
+        o(serviceId?.find('ows:Abstract')?.textContent).equals('the description');
+        o(serviceId?.find('ows:Title')?.textContent).equals('the title');
 
-        const layer = raw.find('Contents', 'Layer')!;
+        o(raw?.find('TileMatrixSetLink')?.toString()).deepEquals(
+            V('TileMatrixSetLink', [V('TileMatrixSet', 'WebMercatorQuad')]).toString(),
+        );
+
+        const layer = raw?.find('Contents', 'Layer');
 
         o(listTag(layer, 'Format')).deepEquals([
-            '<Format>image/png</Format>',
-            '<Format>image/webp</Format>',
-            '<Format>image/jpeg</Format>',
+            V('Format', 'image/png').toString(),
+            V('Format', 'image/webp').toString(),
+            V('Format', 'image/jpeg').toString(),
         ]);
 
-        o(listTag(layer, 'ows:WGS84BoundingBox')).deepEquals([
-            '<ows:WGS84BoundingBox crs="urn:ogc:def:crs:OGC:2:84">\n' +
-                '  <ows:LowerCorner>-180 -85.0511287798066</ows:LowerCorner>\n' +
-                '  <ows:UpperCorner>180 85.0511287798066</ows:UpperCorner>\n' +
-                '</ows:WGS84BoundingBox>',
+        // FIXME I dont think this this really needed? we will need to reproject to get these values
```

Comment:
From memory this is important in some clients such as ArcGIS Desktop (not pro). We should test this out once in dev.

Reply:
Ahh good to know, we are looking to add it back in once we add `proj` to this area of the codebase.